variables.tf

```diff
@@ -21,16 +21,34 @@ variable "enable_account_best_practices" {
 }
 
 variable "nat" {
-  description = "Select NO_NAT for no NAT, NAT_INSTANCES for NAT based on EC2 instances, or NAT_GATEWAY for NAT with AWS NAT Gateways."
+  description = "Select NO_NAT for no NAT, NAT_INSTANCES for NAT based on EC2 instances, NAT_GATEWAY for NAT with AWS NAT Gateways, or ALTER_NAT for hourly based EC2 instances."
```

Comment:
Improve description: "... ALTERN_NAT for using the Open Source project AlterNAT for NAT."